t/upstream_response_time_bugfix.t

```diff
@@ -0,0 +1,35 @@
+# vim:set ft= ts=4 sw=4 et fdm=marker:
+
+use Test::Nginx::Socket::Lua;
+
+repeat_each(1);
+
+plan tests => repeat_each() * (blocks() * 2);
+
+#no_shuffle();
+no_long_string();
+run_tests();
+
+__DATA__
+
+=== TEST 1: ignore the client abort event in the user callback
+--- config
+    location /t {
+        proxy_pass http://httpbin.org/delay/2;
```

Comment:
Better to avoid external services if possible. The external service might make CI unstable.